crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -159,6 +159,81 @@ impl From<IndexeddbCryptoStoreError> for CryptoStoreError {
 
 type Result<A, E = IndexeddbCryptoStoreError> = std::result::Result<A, E>;
 
+/// Defines an operation to perform on the database
+enum PendingOperation {
+    Put { key: JsValue, value: JsValue },
+    Delete(JsValue),
+}
+
+/// A struct that represents all the operations that need to be done to the
+/// database when save_changes is called.
```

Comment:
What is `save_changes`? Can you include backticks and also prefix with the struct/trait holding it?